Neos.Neos/Classes/Command/WorkspaceCommandController.php

```diff
@@ -178,13 +179,23 @@ public function createCommand(
         string $contentRepository = 'default'
     ): void {
         $contentRepositoryId = ContentRepositoryId::fromString($contentRepository);
+
+        $this->workspaceService->createWorkspace(
+            $contentRepositoryId,
+            \Neos\Neos\Domain\Model\WorkspaceTitle::fromString($title),
```

Comment:
should probably be `use`d instead of fq

Reply:
This is a temporary work around, as the `\Neos\ContentRepository\Core\SharedModel\Workspace\WorkspaceTitle` will be replaces in the next step.
But you're right, I should probably rather import the _new_ namespace if the other one is to be removed.

This is just a draft though